docs/user-guide/cli-using-using-daemon-mode.md

```diff
@@ -26,63 +27,93 @@ The following steps describe how to enable daemon mode and how to configure Zowe
 
    ```
    C:\Users\<user_ID>\.zowe\bin
    ```
 
-   **Important!** Ensure that you position the path to your Zowe executables before the path into which NPM installed the Node.js script. For example, `C:\Program Files\nodejs\zowe.cmd`. For information about configuring environment variables, see the documentation for your computer's operating system.
+   :::info Important
+   
+   Ensure that you position the path to your Zowe executables before the path into which NPM installed the Node.js script. For example, `C:\Program Files\nodejs\zowe.cmd`. For information about configuring environment variables, see the documentation for your computer's operating system.
+
+   :::
+
+   You have successfully configured Zowe CLI to run on deamon mode.
+   
+   Each time a Zowe CLI command is issued, the daemon binary creates or reuses a file in the user's home directory. In some cases, this behavior might be undesirable. (For example, the home directory resides on a network drive and has poor file performance.)
+   
+   To change the location that the daemon uses, see [Setting CLI daemon mode properties](../user-guide/cli-configuringcli-ev.md#setting-cli-daemon-mode-properties).
+
+   :::note
    
-   **Alternative configuration**: By default, the daemon binary creates or reuses a file in the user's home directory each time a Zowe CLI command runs. In some cases, this behavior might be undesirable. To change the location that the daemon uses, see [Setting CLI daemon mode properties](../user-guide/cli-configuringcli-ev.md#setting-cli-daemon-mode-properties).
+   Complete the environment variable configuration step (Step 2) only once. **[why would users think they need to perfrom Step 2 multiple times? Is this note really necessary?]**
 
-   **Note:** Complete the environment variable configuration step (Step 2) only once.
+   :::
 
 The following example illustrates running Zowe CLI commands with daemon mode enabled:
 
    ```
    zowe --version
    Starting a background process to increase performance ...
-   7.0.0-next.202110211759
+   8.0.0-next.202110211759
    
    zowe --version
-   7.0.0-next.202110211759
+   8.0.0-next.202110211759
    ```
-**Note:** When you are running Zowe CLI in daemon mode using a Git Bash terminal on a Windows operating system, special characters might display using the wrong code page. For example, the default code page 437 renders a form-feed character (\f) as an emoji (♀️). To correct the problem, issue the command `chcp.com 65001` to change the code page to UTF-8. If you want the change to be persistent, add the command to your `.bashrc` file.
+:::note
+
+When you are running Zowe CLI in daemon mode using a Git Bash terminal on a Windows operating system, special characters might display using the wrong code page. For example, the default code page 437 renders a form-feed character (\f) as an emoji (♀️) **[<--is this really an emoji? do we need a better example?]**.
```

Comment:
@adam-wolfe @zFernand0 - please respond

Reply:
I'm not sure what it should be called. Maybe a Unicode symbol? I guess the question for the example is: What _is_ displayed and what _should be_ displayed?

Reply:
This note can be removed.

Even though the lower range (1 - 31 & 127) are contextual, I did not experience any difference on Git Bash because of the default codepage (437).

This Stackoverflow answer helped me understand (or confuse myself) a bit more 😋 
- https://stackoverflow.com/a/45377018